flutter/Graminsta/lib/post/post_widget.dart

```diff
@@ -0,0 +1,122 @@
+import 'package:Graminsta/config.dart';
+import 'package:Graminsta/models/post.dart';
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+
+Widget postWidget(Post post) {
+  String _markedUser = '';
```

Comment:
No need for the underscore, as this is a local variable and can't be accessed from outside this function.